server/sonar-server/src/main/java/org/sonar/server/component/ws/ComponentDtoToWsComponent.java

```diff
@@ -57,6 +57,9 @@ private ComponentDtoToWsComponent() {
     setNullable(emptyToNull(dto.language()), wsComponent::setLanguage);
```

Comment:
I'm now using `SONAR-9096 add visibility to response of api/components/search_projects` (71 chars)